src/lib/apiTypesV2/Subscription.cpp

```diff
@@ -42,7 +45,8 @@ std::string Subscription::toJson()
   jh.addString("id", this->id);
   if (!this->duration.isEmpty())
   {
-    jh.addRaw("duration: ",this->duration.render(JSON,"", false));
+    //jh.addRaw("duration: ",this->duration.render(JSON,"", false));
```

Comment:
I needed to change this due to the current code was producing a wrong JSON token:
```
"duration: ":
```

Maybe it is a way of using the Duration render() that fix it, without needing to composed using the duration.get().